services/config/src/types.ts

```diff
@@ -22,3 +22,7 @@ export interface Config {
     serverVersion?: Version
     systemInfo?: SystemInfo
 }
+
+export interface ConfigWithSkipApiVersion extends Config {
```

Comment:
The extra props provided to provider are actually already being passed downwards (https://github.com/dhis2/app-runtime/blob/master/services/data/src/react/components/DataProvider.tsx#L36-L39)

but the type definition used (e.g. https://github.com/dhis2/app-runtime/pull/1374/files#diff-accda02931529da3af34ae8dbe9c0ea5549c444fdbcaf5a7d504a086dfd9331eR76) caused problems, so I'm extending it. 

I'm not sure what the logic was of originally passing the props as part of the config, so I haven't updated things any more.